app/src/main/java/org/ole/planet/myplanet/MainApplication.kt

```diff
@@ -70,6 +72,9 @@ class MainApplication : Application(), Application.ActivityLifecycleCallbacks {
 
         context = this
         preferences = getSharedPreferences(PREFS_NAME, MODE_PRIVATE)
+        nightMode()
+        // UNCOMMENT BELOW TO FORCE DARK MODE FOR DARK MODE DEVELOPMENT
+        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES)
```

Comment:
If you wanted to test the light mode functionality, you will have to comment this line out. (I would actually recommend doing this as you can still switch to dark mode functionality via settings if this line is commented out).